controller/websocket_controller.go

```diff
@@ -69,10 +69,16 @@ func NewWebsocketController(
 	wc.cacheSyncs = append(wc.cacheSyncs, ds.BackupInformer.HasSynced)
 	ds.RecurringJobInformer.AddEventHandler(wc.notifyWatchersHandler("recurringJob"))
 	wc.cacheSyncs = append(wc.cacheSyncs, ds.RecurringJobInformer.HasSynced)
+	ds.ObjectStoreInformer.AddEventHandler(wc.notifyWatchersHandler("objectstore"))
```

Comment:
Can we use `objectStore` instead of `objectstore` to be uniformed with other resources' naming

Reply:
ping @m-ildefons